shared/package.json

```diff
@@ -150,27 +144,28 @@
     "eslint-plugin-promise": "3.5.0",
     "eslint-plugin-react": "7.0.1",
     "eslint-plugin-standard": "3.0.1",
-    "express": "4.15.3",
     "file-loader": "0.11.1",
     "flow-bin": "0.41.0",
-    "fs-extra": "3.0.1",
+    "fs-extra": "git://github.com/keybase/node-fs-extra#80cec860b639db7b6c11893615b7f6495a82b985",
     "jest": "20.0.4",
     "minimist": "1.2.0",
     "null-loader": "0.1.1",
     "pegjs": "0.10.0",
     "react-addons-perf": "15.4.2",
-    "react-hot-loader": "3.0.0-beta.6",
+    "react-hot-loader": "next",
```

Comment:
I'm fine making that change but I wonder if there's real value in doing it post yarn. In my updating flow I use yarn outdated which didn't actually pick up this being outdated. Maybe a bug with betas or something.